src/Helper/Helper_Abstract_Options.php

```diff
@@ -1545,10 +1546,13 @@ public function multicheck_callback( $args ) {
 				$args['multi-option'] = $key;
 
 				$enabled = $this->get_form_value( $args );
+				$label = esc_html( $option ) ;
+				$aria_checked = !empty( $enabled ) ? 'aria-checked="true"' : 'aria-checked="false"';
+				$aria_label = str_replace( '_', ' ',$label );
 
 				echo '<div class="gfpdf-settings-multicheck-wrapper">';
-				echo '<input name="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" id="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" class="gfpdf_settings_' . $args['id'] . ' ' . $class . '" type="checkbox" value="' . $args['multi-key'] . '" ' . checked( $key, $enabled, false ) . ' ' . $required . ' />&nbsp;';
-				echo '<label for="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']">' . esc_html( $option ) . '</label>';
+				echo '<input name="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" id="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" class="gfpdf_settings_' . $args['id'] . ' ' . $class . '" type="checkbox" value="' . $args['multi-key'] . '" ' . checked( $key, $enabled, false ) . ' ' . $required . ' role="checkbox" ' . $aria_checked . ' />&nbsp;';
+				echo '<label for="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" aria-label="'. $aria_label .'">' . $label . '</label>';
```

Comment:
All my reading suggests that `aria` tags aren't necessary on a `<label>` tag with a correctly referenced `for` target:

https://stackoverflow.com/questions/55814279/aria-label-vs-form-labels
https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Could you please provide reference martial that backs up why using both is appropriate?

Reply:
As for my test Jake. aria-label is prioritized by screen reader. My goal was to design the labels to more user friendly in terms of audio. Thus $aria_label = str_replace( '_', ' ',$label ); e.g gravityforms_view_entries will be read as gravityforms view entries.
Anyways this item is still experimental.

Reply:
That makes sense, but we are talking about specific capability names [which are described in detail in the Gravity Forms documentation](https://docs.gravityforms.com/role-management-guide/). If they had read the GF docs, users would be looking for `gravityforms*underscore*view*underscore*entries`, rather than `gravityforms view entries`.